### ISSUE TITLE
Version 1.1.7 (118) — crash

### ISSUE THREAD

**Issue by jeff-h** [eofster/Telephone#245]
I updated to 1.1.7 recently via the App Store app. The new version opens up fine, but as soon as it receives an incoming call request (i.e. would usually start ringing) and as soon as I initiate an outgoing call (when the other phone starts ringing) it locks up with a beachball.

The console reports:

```
2/07/16 2:11:34.700 PM Telephone[93928]: [Accounts] Failed to update account with identifier B213BFF5-56BA-4BF9-9E32-2894ECEEF85B, error: Error Domain=ABAddressBookErrorDomain Code=1002 "(null)"
```

Let me know if there's anything I can do to help narrow this down... or is it likely a broader system issue on my Mac?

**Comment by jeff-h** [comment]
(I tried deleting the app and re-downloading from the App Store in case the update process had gone wrong, but there was no change).

**Comment by eofster** [comment]
Could you attach the full crash report? Does Telephone have access to system Contacts (System Preferences > Security & Privacy > Privacy > Contacts > Telephone)?

**Comment by jeff-h** [comment]
I checked where you suggested and Telephone does have access to my system contacts.

However, the problem appears to have gone away. Two things have changed, so unfortunately I can't say which it was that appears to have fixed it.

* I turned off Contacts access, quit Telephone, then turned on Contacts access again and relaunched Telephone (but ran out of time to test if fixed the crash).

* then my Mac crashed and restarted itself during the night for an unknown reason. It's never done this before, but I don't see any reason to suspect Telephone as the cause.

So now it works :) Thanks for your time responding. Hopefully it's just a weird one-off problem.
